hw/formal/fpv.tcl

```diff
@@ -14,10 +14,10 @@ set_message -disable VERI-2418
 #-------------------------------------------------------------------------
 # read design
 #-------------------------------------------------------------------------
+
 analyze -sv09 \
   -f formal_0.scr \
-  [glob ../../../../ip/*/dv/*_bind.sv] \
-  [glob ../../../../ip/*/dv/tb/*_bind.sv] \
+  [glob ../../../../ip/$env(FPV_TOP)/dv/*_bind.sv ../../../../ip/$env(FPV_TOP)/dv/tb/*_bind.sv] \
```

Comment:
Perhaps having two lines as was done before is more readable and avoids long line length?

Reply:
I think the script also consider the dependent bind files too. If IP A has submodule B, then it should include B_bind.sv too. The original script was considered that even it reads all bind files.

Reply:
Thanks for your suggestion Nils, I was using the two lines before, but if some module does not have the specific items, say rv_plic does not have `../../../../ip/$env(FPV_TOP)/dv/*_bind.sv`, then it will throw an error. But you are right - it does not look that elegant.
Thanks Eunchan for the suggestion too, I think the dependency should base on the core file. But that is a good point, Michael is proposing a different way to handle this. I will get back to this PR after discuss with him.

Reply:
I think you can use the -nocomplain so that it doesn't error out when it can't find a file (I believe I'm using this switch in one of the previous lint scripts)